platform/http_uploader_apple.mm

```diff
@@ -10,11 +7,93 @@
 
 #include <memory>
 
+@interface MultipartUploadTask : NSObject
+
+@property (nonatomic, copy) NSString *method;
```

Comment:
please use our code style about property (you can check it in our project).
for example here is 
```objc
 @property(copy, nonatomic) NSString * method;
```